operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -833,6 +871,7 @@ public String getModelHome() {
         .map(Configuration::getModel).map(Model::getModelHome).orElse(DEFAULT_WDT_MODEL_HOME);
   }
 
+  // todo - should these be put into alphabetical order? It is getting increasingly difficult to find items
```

Comment:
They were originally in the order that they were defined in the class. I don't think there is any technical reason to oppose that they be alphabetically sorted.

Reply:
I'm alphabetizing `toString` and `hashCode` but not `equals` for now, as the latter, unlike the first two, mixes field references with getters, and I don't know why that was done.